### DISCUSSION TITLE
part-ii/2019-120/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#109] [Question comments]
Part II 2019 Probability and Measure: Paper 1, Section II, K 

https://questions.tripos.org/part-ii/2019-120/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(a) For a straightforward integration, can calculate $\mathbb{E}\left[e^{uX}\right]$, and substitute $u$ with $iu$, using analyticity of both sides. The bookwork proof uses differentiation under the integral sign, which avoids this problem. 

(b) 

(i) characteristic function is real if distribution is symmetric

(ii) Pythagoras

(iii) characteristic function is bounded by 1. 

(iv) Levy's criterion

</details>

**Reply by ArthurConmy** [reply]
Pretty sure you’re supposed to just quote the 1D characteristic function and then easily derive the multidimensional one, nothing interesting required
